opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/bmp.adoc

```diff
@@ -151,30 +152,34 @@ persisted in OpenNMS Postgres DB.
 |===
 
 Stats and retrieving data from external data for rpki/routeinfo/asinfo are handled only on {opennms-product-name}.
-For this `opennms-telemetry-bmp-stats` feature need to be installed on {opennms-product-name}.
+The `opennms-telemetry-bmp-stats` feature needs to be installed on {opennms-product-name}.
+
+BMP Stats are stored as time series data in Postgres with TimescaleDB extension.
```

Comment:
@Bonrob2 I missed adding TimescaleDB related info to docs in previous commit. Added here.

Reply:
LGTM!